src/simul.c

```diff
@@ -1561,6 +1562,7 @@ void simul_dump() {
       if (dump_delay_png && (time_threshold == time_threshold_dpng)) {
         //dump PNG file
         sprintf(name, "%s%05d_t0%s.png", MOD_NAME, cpt_dump, str);
+        output_path_noext(name);
```

Comment:
@defaziogiancarlo : Does this conflict with your work?

Reply:
It outputs images and .csp.gz files to the directory specified by the params file.

Reply:
In cellspace.py there's an assumption that the .csp files are in the same directory as the run script. That's an easy fix, I'll change it.